kasa/smartstrip.py

```diff
@@ -154,6 +155,35 @@ async def set_alias(self, alias: str) -> None:
         """
         return await super().set_alias(alias)
 
+    @requires_update
+    async def get_emeter_realtime(self) -> EmeterStatus:
+        """Retrieve current energy readings."""
+        if not self.has_emeter:
+            raise SmartDeviceException("Device has no emeter")
+
+        plug_emeter = False
+        for plug in self.children:
+            if plug.has_emeter:
+                plug_emeter = True
+
+        if not plug_emeter:
+            return EmeterStatus(await self._query_helper(self.emeter_type, "get_realtime"))
+
+        emeter_rt: DefaultDict[int, float] = defaultdict(lambda: 0.0)
+        count = 0
+        for plug in self.children:
+            if not plug.has_emeter:
+                continue
+            count += 1
+            plug_emeter_rt = await plug.get_emeter_realtime()
+            for field, value in plug_emeter_rt.items():
+                emeter_rt[field] += value
+    
+        # Voltage is averaged
+        emeter_rt['voltage_mv'] /= count
```

Comment:
I get the idea here, but have you ever noticed the individual plugs reporting different voltage? Normally you should get the same voltage from each child-plug.